src/iframe.php

```diff
@@ -32,18 +32,29 @@ public function lazyloadIframes($html, $args)
                 continue;
             }
 
-            if ($args['youtube'] && false !== strpos($iframe[1], 'youtube')) {
-                $youtube_lazyload = $this->replaceYoutubeThumbnail($iframe);
+            // Given the previous regex pattern, $iframe['atts'] starts with a whitespace character.
+            if (! preg_match('@\ssrc\s*=\s*(\'|")(?<src>.*)\1@iUs', $iframe['atts'], $atts)) {
+                continue;
+            }
+
+            $iframe['src'] = trim($atts['src']);
+
+            if ('' === $iframe['src']) {
+                continue;
+            }
+
+            if ($args['youtube'] && false !== strpos($iframe['src'], 'youtube')) {
```

Comment:
I removed this `strpos()`. The reason is because in `getYoutubeIDFromURL()` we match other things than simply "youtube": `(?:youtu\.be|youtube\.com|youtube-nocookie\.com)`.

In this case I would simplify everything like this:
```php
if ($args['youtube']) {
    $iframe_lazyload = $this->replaceYoutubeThumbnail($iframe, $iframe['src']);
}

if (empty($iframe_lazyload)) {
    $iframe_lazyload = $this->replaceIframe($iframe, $iframe['src']);
}

$html = str_replace($iframe[0], $iframe_lazyload, $html);
```